webapi/src/main/scala/org/knora/webapi/routing/v1/SearchRouteV1.scala

```diff
@@ -74,6 +76,9 @@ object SearchRouteV1 extends Authenticator {
             case other => None
         }
 
+        //
+        // the params with multiple values have angle brackets appended: param name[]
+        //
```

Comment:
Do these params still have brackets appended? I thought the GUI was changed to remove them. Also, [] are called square brackets. Angle brackets are like this: <>.